Apps/GatewayApi/test/unit/Controllers.Test/UserProfileControllerTests.cs

```diff
@@ -511,8 +523,8 @@ private static Mock<IHttpContextAccessor> CreateValidHttpContext(string token, s
         {
             IHeaderDictionary headerDictionary = new HeaderDictionary
             {
-                { "Authorization", token },
-                { "referer", "http://localhost/" },
+                {"Authorization", token},
+                {"referer", "http://localhost/"},
             };
```

Comment:
Nit: something might be off with your ReSharper settings because those spaces are supposed to be there (and they aren't removed when I run the reformat and cleanup code action).

Reply:
Resolved.